### PR TITLE
Merge remote-tracking branch 'refs/remotes/oliversaal/master'

### PR DESCRIPTION
**Pull request by WaydeTheunissen**
docs folder and images updated

### CODE REVIEW

**Comment by oliversaal** [comment]
I don't see any diffs in this PR. Did you submit it correctly? Even though you forked TL, I don't see the commit you made to the docs or images folder. All I see is the last commit we made to the branch. Can you see your commit changes in the PR?

File: docs/WITW_v2.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,user-scalable=no">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="Digital Art Thingy Inc. explores cross platform development opportunities on the desktop and mobile devices.">
+    <meta name="author" content="Digital Art Thingy Inc.">
+    <link rel="icon" href="favicon.ico">
+
+    <title>Where in the world ...</title>
+
+    <!-- Bootstrap core CSS -->
```
Comment (oliversaal) [inline docs/WITW_v2.html]: I've been wondering - do we really need to host our own bootstrap install? Could we not point to some popular CDN version and remove all these bootstrap files? Any non-standard bootstrap files not available on the CDN and still exist locally.

File: docs/WITW_v2.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            <li>Direction - Lost and need a quick reference point to where you are.</li>
+            <li>Educational - Use the map to see your neighboring countries, states and provinces in relation to your current position.</li>
+            <li>Share your location - Take a screenshot while using the app an send it to a friend or family member.</li>
+        </ul>
+        
+        <p>Our motivation for creating this app was to learn how to incorporate maps into an mobile app. Learn with us. Our code is open source. Visit our GitHub repository clone, comment or report issues.</p>
+        
+        <p>Watch this space for more apps from Digital Art Thingy Inc.</p>
+      </div>
+
```
Comment (oliversaal) [inline docs/WITW_v2.html]: I noticed this in the original HTML file as well - can we remove the unecessary new lines and maintain the tight consistency we employed in other <div></div> blocks. My eyes are just drawn to inconsistencies ;-)

File: docs/WITW_v2.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          </button>
+          <a class="navbar-brand" href="#">Where in the world ...</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav">
+            <li class="active"><a href="#">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div><!--/.nav-collapse -->
```
Comment (oliversaal) [inline docs/WITW_v2.html]: I think we should get rid of inline comments in HTML. I prefer comments on their own line if we have them.

File: docs/WITW_v2.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      </div>
+    </nav>
+
+    <div class="container theme-showcase main_container" role="main">
+    	<section class="focus_block">
+    		<img src="images/Alpha_Geo-Location_Feature_Graphic.png" alt="" />
+    	</section>
+      <!-- Main jumbotron for a primary marketing message or call to action -->
+      <div class="jumbotron">
+        <p>Where in the world is a simple location aware app. How you decide to use it is up to you. Here are some ideas:</p>
```
Comment (oliversaal) [inline docs/WITW_v2.html]: Maybe we should really just say it is: "Where in the World is a Google Maps tutorial implemented in a simple Android application. Eventually we're going to fork this app and create a more useful location awareness app.

File: docs/WITW_v2.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        	<div class="image_container">
+				<img src="images/Alpha_Map_Screenshot_Phone.jpg" alt="" />
+            </div>
+        <ul>
+            <li>Travel - Explore the map and plan your adventures.</li>
+            <li>Direction - Lost and need a quick reference point to where you are.</li>
+            <li>Educational - Use the map to see your neighboring countries, states and provinces in relation to your current position.</li>
+            <li>Share your location - Take a screenshot while using the app an send it to a friend or family member.</li>
```
Comment (oliversaal) [inline docs/WITW_v2.html]: TBD? That's not supported yet - should we mention it or mark it as *Coming soon*

File: docs/WITW_v2.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        </ul>
+        
+        <p>Our motivation for creating this app was to learn how to incorporate maps into an mobile app. Learn with us. Our code is open source. Visit our GitHub repository clone, comment or report issues.</p>
```
Comment (oliversaal) [inline docs/WITW_v2.html]: app --> application

Comment (oliversaal) [inline docs/WITW_v2.html]: It's also a long line - Chrome is not offering me a horizontal scrollbar :-( 

Did we mention the GitHub repository location or should we omit that sentence and just indicate that the application was our first Android application exploring the Google Maps APIs.

File: docs/WITW_v2.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+        
+        <p>Watch this space for more apps from Digital Art Thingy Inc.</p>
```
Comment (oliversaal) [inline docs/WITW_v2.html]: apps --> applications

File: docs/WITW.html [hunk 8]
```diff
@@ -1,608 +1,88 @@
...
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+
+    <div class="container theme-showcase main_container" role="main">
+    	<section class="focus_block">
+    		<img src="images/Beta_Geo-Location_Feature_Graphic.png" alt="" />
+    	</section>
```
Comment (oliversaal) [inline docs/WITW.html]: This is nitpicky but I prefer if we keep all asset filenames (i.e. JPGs, CSS, etc) lowercase. Especially if someone forks this repo on a non-Windows machine, it's easy to forget "Beta*" != "beta*". 

I also mentioned this to Nicholas but we should probably not use the WITW shorthand in the assets. It's okay at our code level but we have the real estate space in our images to use the full "Where in the world..." title.

File: docs/theme.css [hunk 9]
```diff
@@ -1,18 +1,15 @@
...
-}
-
-.theme-showcase > p > .btn {
-  margin: 5px 0;
-}
-
-.theme-showcase .navbar .container {
-  width: auto;
-}
+body { padding-top: 70px; padding-bottom: 30px;}
```
Comment (oliversaal) [inline docs/theme.css]: Nit, can we follow the same {} (parenthesis) convention used in the other CSS files. Brackets on their own line.

```
.main_container {
    padding: 0;
}
```
over the line version: 

.main_container { padding: 0; }

Just for my education, why do you remove the "px" or "%" suffixes?

File: docs/WITW_v2.html [hunk 10]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
Comment (oliversaal) [inline docs/WITW_v2.html]: What is the difference between WITW_v2.html and WITW.html? Can we get rid of one of them and just default to WITW.html? I didn't see any changes to index.html so I assume WITW_v2.html is unreferenced.

File: docs/WITW_v2.html [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+
+    <div class="container theme-showcase main_container" role="main">
+    	<section class="focus_block">
+    		<img src="images/Alpha_Geo-Location_Feature_Graphic.png" alt="" />
```
Comment (oliversaal) [inline docs/WITW_v2.html]: Nit:

I noticed in Nicholas's original images we subtitled WITW with "The Geo-Location app". I don't know if we need to do that. Just call it "Where in the world..." It's intuitive enough that it's a location app ala Google Maps. Most people know what Google Maps does. We shouldn't try and use too many location-centric descriptions e.g. location awareness, geo-location, GoogleMaps, etc. If you use Google Maps, you also don't need to worry about localization of these terms if you ever wanted to translate it into other languages.

**Comment by oliversaal** [comment]
Nic and I also went back and forth about our checkin process. We "SQUASH AND MERGE" after the review is signed off but we also make sure to title the commit/checkin with a brief yet descriptive title. We try and avoid including branch names or users in the title. See the other commits in TL for example.

File: docs/WITW_v2.html (hunk 4 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: This was the description that I asked Wayde to implement.  I'm fine with it being changed.

File: docs/WITW_v2.html (hunk 5 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: These are some examples of how the app can be used today. Technically it's all possible. My intention was to make the app sound exciting. We want them to download it right?

File: docs/WITW_v2.html (hunk 6 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: I'm leaning towards adding the GitHub repository location.

File: docs/WITW_v2.html [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,user-scalable=no">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="Digital Art Thingy Inc. explores cross platform development opportunities on the desktop and mobile devices.">
+    <meta name="author" content="Digital Art Thingy Inc.">
+    <link rel="icon" href="favicon.ico">
+
+    <title>Where in the world ...</title>
```
Comment (nicsaal) [inline docs/WITW_v2.html]: Did we not drop the ellipses points from our app title? Also the "W" in world should be capitalized.

File: docs/WITW_v2.html (hunk 11 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: Are you referring to the image name? I named the images this to score extra SEO points. I figured the more we mention geo location in the html markup the higher the website will rank in a search engines.

File: docs/WITW_v2.html (hunk 5 above)
Comment (oliversaal) [inline docs/WITW_v2.html]: We shouldn't oversell or promise features that we haven't implemented yet. Disappointed users will downrate the app in the store and the ability to change their rating after their initial use is harder. Users trust the store rating from other users more than they trust the developers upsell website.

File: docs/WITW_v2.html (hunk 11 above)
Comment (oliversaal) [inline docs/WITW_v2.html]: I'm referring to the title in the image (not just the filename). "WITW" is a shorthand or abbreviation that people will not naturally use. We don't have that level of branding *yet*. 

We don't need to do target SEO in our app: we're pretty much building something for Google and they've given us a direct pipeline to being picked up in their search engine/store. Don't bend over backwards for this feature.

File: docs/WITW_v2.html (hunk 12 above)
Comment (oliversaal) [inline docs/WITW_v2.html]: Your call! I still like using the ellipsis. It doesn't hurt that we give users the impression our app is not specifically targeting places/people/landmarks but possibly all or many more.

File: docs/WITW_v2.html (hunk 11 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: Ok, I will amend the graphic and push a new PR

File: docs/WITW_v2.html (hunk 12 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: Keep the Ellipses. I checked the app and it has the ellipses too. I thought we took it out but was mistaken.

File: docs/WITW_v2.html (hunk 5 above)
Comment (nicsaal) [inline docs/WITW_v2.html]: I see, in that case remove it from the page.

File: docs/WITW_v2.html (hunk 10 above)
Comment (oliversaal) [inline docs/WITW_v2.html]: Any progress - removing WITW_v2 or rolling it into WITW.html?

File: docs/WITW.html (hunk 8 above)
Comment (oliversaal) [inline docs/WITW.html]: I guess no change on the casing?

**Pull request merged**